app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,48 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.SessionRequestData;
+import com.codesoom.assignment.dto.UserLoginData;
+import com.codesoom.assignment.errors.InvalidAccessTokenException;
+import com.codesoom.assignment.errors.InvalidTokenException;
+import com.codesoom.assignment.utils.JwtUtil;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.security.SignatureException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+/***
+ * 사용자 인증 관련 서비스 로직
+ */
+@Service
+public class AuthenticationService {
+
+    private UserService userService;
+    private JwtUtil jwtUtil;
+
+    public AuthenticationService(JwtUtil jwtUtil, UserService userService) {
+        this.userService = userService;
+        this.jwtUtil = jwtUtil;
+    }
+
+    /**
+     * 로그인 처리 로직
+     *
+     * @return JWT
+     */
```

Comment:
리턴값 위주로 설명해주면 코드의 내용을 읽지 않아도 3초 만에 이 메소드가 뭐하는 메소드인지 파악할 수 있습니다.
```suggestion
    /**
     * 사용자를 로그인 처리하고, 인증 토큰을 생성해 리턴합니다.
     * @param SessionRequestData 로그인 요청 정보
     * @return 인증 토큰
     */
```